kernel/device.c

```diff
@@ -9,6 +9,10 @@
 #include <sys/atomic.h>
 #include <syscall_handler.h>
 
+#define LOG_LEVEL CONFIG_KERNEL_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(device);
```

Comment:
Setting `LOG_LEVEL` is not really needed. It works, but you can say
`LOG_MODULE_REGISTER(device, CONFIG_KERNEL_LOG_LEVEL);` instead too.